lib/main.coffee

```diff
@@ -7,38 +7,75 @@ module.exports =
 
   config:
     compatMode:
+      title: 'Compatibility mode (AsciiDoc Python)'
       type: 'boolean'
-      default: true
+      default: false
+      order: 1
+    forceExperimental:
+      title: 'Force enable experimental extensions'
+      description: '''
+        The features behind this attribute are subject to change and may even be removed in a future version.
+
+        Currently enables the UI macros (`button`, `menu` and `kbd`).
+        '''
+      type: 'boolean'
+      default: false
+      order: 2
     showTitle:
+      description: '''
+        If set, displays an embedded document’s title.
+
+        Mutually exclusive with the notitle attribute.
+        '''
       type: 'boolean'
       default: true
+      order: 3
     safeMode:
+      description: '''
+        Set safe mode level: `unsafe`, `safe`, `server` or `secure`.
+
+        Disables potentially dangerous macros in source files, such as `include::[]`.
```

Comment:
Maybe we should explain how to allow include macros ?
"Set safe mode level to safe to enable include macros, but restricts access to ancestor paths of source file."